### PR TITLE
[CLI] Harden labeler startup and audit flow

### PR DESCRIPTION
**Pull request by EricCogen**
## Summary

Hardens the corpus labeler startup and audit flow to handle edge cases that caused crashes or silent failures during large labeling runs.

## Changes

- Improved labeler startup validation
- Added guards around audit flow edge cases
- Better error surfacing on initialization failures

## Testing

All existing tests pass.

### CODE REVIEW

File: src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py [hunk 1]
```diff
@@ -1,484 +1,512 @@
...
+        if pending > 0 or in_progress > 0:
+            print(
+                f"[startup] Queue already populated. Pending={pending}, in_progress={in_progress}, total={total}. Skipping seed."
+            )
+            return
+
+        total = seed_queue(conn)
+        print(f"[startup] Queue seeded. Total tasks in label_queue: {total}")
+
+
```
[the remark this answers is not in the document]
Comment (EricCogen) [reply src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py:84]: Acknowledged. The startup guard is intentional: re-seeding while work is in progress would duplicate tasks and confuse the labeler UI. INSERT OR IGNORE prevents duplicates but the skip avoids the overhead entirely. The guard stays.

File: src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py [hunk 2]
```diff
@@ -1,484 +1,512 @@
...
+def get_conn():
+    conn = connect(CONFIG["database_path"])
+    init_app_tables(conn)
+    return conn
+
+
+def _should_run_startup_seed() -> bool:
+    # Flask's debug reloader starts a parent process and then the real serving process.
+    # Only seed in the serving process to avoid paying startup cost twice.
+    return not DEBUG or os.environ.get("WERKZEUG_RUN_MAIN") == "true"
```
[the remark this answers is not in the document]
Comment (EricCogen) [reply src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py]: Fixed in d06b7ba. DEBUG now reads FLASK_DEBUG env var (defaults off). The reloader guard uses the runtime DEBUG value so it correctly tracks the actual debug mode.

File: src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py [hunk 3]
```diff
@@ -1,484 +1,512 @@
...
+
+BASE_DIR = Path(__file__).resolve().parent
+CONFIG = load_config(BASE_DIR)
+
+app = Flask(__name__)
+app.secret_key = CONFIG["secret_key"]
+DEBUG = True
+
+# Resolve relative to repo root (two levels up from app.py)
+_RULES_MD = Path(__file__).resolve().parent.parent.parent.parent / "docs" / "rules.md"
```
[the remark this answers is not in the document]
Comment (EricCogen) [reply src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py]: Fixed in d06b7ba. Comment updated to 'four levels up from app.py' to match the actual parent.parent.parent.parent traversal.

File: src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py [hunk 4]
```diff
@@ -1,484 +1,512 @@
...
+        pending_count=pending_count,
+        item_suggestion=item_suggestion,
+        linked_issues=linked_issues,
+    )
+
+
+@app.post("/task/<int:queue_id>/label")
+def submit_label(queue_id: int):
+    decision = request.form.get("decision", "").strip()
+    confidence = float(request.form.get("confidence", "0.75") or 0.75)
```
[the remark this answers is not in the document]
Comment (EricCogen) [reply src/GauntletCI.Corpus/GauntletCI_Labeler_App/app.py]: Fixed in d06b7ba. confidence now has try/except ValueError with fallback to 0.75, matching the auto_label_apply() pattern.
